tests/integration_tests/test_examples.py

```diff
@@ -16,15 +16,16 @@
 ]
 
 LONG_RUNNING_EXAMPLES = [
-    EXAMPLES_ROOT_DIR / 'tf/ppo_memorize_digits.py',
-    EXAMPLES_ROOT_DIR / 'tf/dqn_pong.py',
-    EXAMPLES_ROOT_DIR / 'tf/trpo_cubecrash.py',
+    EXAMPLES_ROOT_DIR / 'tf/ppo_memorize_digits.py', EXAMPLES_ROOT_DIR /
```

Comment:
@ahtsan the formatting looks weird here. I'm approving the PR but please fix this before merging.

Reply:
Thanks. Fixed